Khan/perseus
# Correct how arrowhead rotation is calculated

## Summary:
Arrowhead rotation (aka "direction") was calculated using graph coordinates. While this works properly most of the time, there are situations where it doesn't. Specifically, when grid units are displayed more rectangularly than as a regular square. This code change resolves that issue by using pixel coordinates, instead.


Issue: LEMS-1996

## Test plan:
1. Launch the Dev environment (`yarn dev`)
1. Go to the flipbook (http://localhost:5173/#flipbook)
1. Paste in the following graph definition:
```
{"content":"[[☃ interactive-graph 1]]\n\n\n","images":{},"widgets":{"input-number 1":{"alignment":"default","graded":true,"options":{"answerType":"number","inexact":false,"maxError":0.1,"simplify":"required","size":"small","value":"2.6666666666666665"},"type":"input-number","version":{"major":0,"minor":0}},"interactive-graph 1":{"alignment":"default","graded":true,"options":{"backgroundImage":{"bottom":0,"height":0,"left":0,"scale":1,"url":null,"width":0},"correct":{"coords":[[0,0],[3,8]],"type":"linear"},"graph":{"type":"linear"},"gridStep":[1,1],"labels":["t","d"],"markings":"graph","range":[[-10,1],[-10,10]],"rulerLabel":"","rulerTicks":10,"showProtractor":false,"showRuler":false,"snapStep":[0.5,0.5],"step":[1,1]},"type":"interactive-graph","version":{"major":0,"minor":0}}}}
```
4. Move one of the line points around and note that the arrowheads appear to align properly with the line.

## Affected behavior:

### Before
![Arrowhead Rotation - Before](https://github.com/Khan/perseus/assets/13896410/7bb35816-67df-4bb1-af18-eae175bd1e06)


### After
![Arrowhead Rotation - After](https://github.com/Khan/perseus/assets/13896410/268bec3a-2170-4f2f-8455-ca96befa7c2b)
